doc/manual/manual.tex

```diff
@@ -9474,6 +9474,211 @@ \subsubsection{Yamauchi \& Takei anelastic shear wave velocity-temperature conve
 
 The parameter file and initial temperature model for this benchmark can be found at \url{benchmarks/yamauchi_takei_2016_anelasticity/yamauchi_takei_2016_anelasticity.prm} and \url{benchmarks/yamauchi_takei_2016_anelasticity/anelasticity_temperature.cc}. Code to recreate  Fig.~\ref{fig:anelasticity}  is provided in \url{benchmarks/yamauchi_takei_2016_anelasticity/plot_output}.
 
+\subsubsection{Thin shell gravity benchmark}
+\label{sec:benchmark-thin-shell-gravity}
+
+\textit{This section was contributed by Cedric Thieulot, Bart Root and Paul Bremmer.}
+
+The domain is a spherical shell of 10~\si{\km} radius centered at a depth $D$, i.e. the inner radius is $R_i=6371-D-5~\si{\km}$ and the outer radius $R_o=6371-D+5~ \si{\km}$. It is filled with a fluid of constant density $\rho_0=3300~\si{\kg\per\cubic\metre}$ with total mass $M=\frac43 \pi (R_o^3-R_i^3)$.
```

Comment:
In the manual, subscripts that aren't indices are typically in roman typeface. The easiest way to do this is, for example: `$R\textsubscript{inner}=6371-D-5~\si{\km}$`. Same for `outer`, rather than `$o$`

Reply:
Separately, would you mind adding a short paragraph at the top of this section that starts it out by saying what the goal here is? You're jumping right into the weeds :-)

Reply:
did that

Reply:
also: fun fact, M is wrong here :)